### DISCUSSION TITLE
Video thumbnailers not working after upgrade from LMDE5 to LMDE6

### DISCUSSION THREAD

**Discussion by Jeremy7701** [linuxmint/discussions#185] [Questions] [answered]
All other thumbnailing appears OK.
Did some digging.

`/usr/share/thumbnailers/` has **3** thumbnailers which can process a mimetype of `video/mp4` (and many other mimetypes too) ;-
ffmpegthumbnailer.thumbnailer
totem.thumbnailer
xplayer.thumbnailer

**ffmpegthumbnailer.thumbnailer was missing its trailing semi-colon, but fixing this made no difference.**

I don't know which thumbnailer gets priority for a duplicated mime type?
Presumably I can't just disable a couple of thumbnailers to make one work - there are loads of antique mime-types in there!

**Comment by Jeremy7701** [comment, marked as answer]
Since the LMDE6 media didn't come with a `totem.thumbnailer` file when it was loaded into a VM (and video thumbnailing works there), I tried disabling the totem.thumbnailer on the live machine - and video thumbnailing works again.

So I'd probably call this a BUG fix - but note that since **totem** is a GNOME video player, the presence of a thumbnailer isn't that unexpected.

**Comment by mtwebster** [comment]
Xplayer has been retired except for translation updates for quite a while now, not sure why you still need it - there should be no conflict, just ffmpeg

As far as I know, based on other similar types of plugins, the priority is based on the order the thumbnailer definition files are read from the disk - from the function  that reads the dirs: `The order of entries returned from this function is not defined, and may vary by file system or other operating-system dependent factors.`/

This isn't something we can do anything about, as the mechanism can't know whatever it chooses is malfunctioning until it tries to use it.

**Comment by Jeremy7701** [comment]
I tried re-enabling `/usr/share/thumbnailers/totem.thumbnailer` and disabling `xplayer.thumbnailer` (since I have no xplayer installed).
### Video thumbnailing didn't work.
So I returned to the previous situation of using
ffmpegthumbnailer.thumbnailer
**totem.thumbnailer** - renamed as `totem.thumbnailer-disabled`
xplayer.thumbnailer

This worked correctly - note that whilst I have no xplayer installed, I do have `xplayer-video-thumbnailer` installed.
**I appreciate that the order of thumbnailers isn't known.**
